packages/data/src/ethers.ts

```diff
@@ -17,8 +17,8 @@ import {
     Provider
 } from "ethers/providers"
 import { SemaphoreABI } from "@semaphore-protocol/utils/constants"
+import { errorHandlers } from "@zk-kit/utils"
```

Comment:
```suggestion
import { requireString } from "@zk-kit/utils/error-handlers"
```

The module can be imported directly. This way it's no longer necessary to use `errorHandlers.requireString` and `requireString` can be used directly.